lib/libds3/src/pointers.rs

```diff
@@ -234,6 +235,7 @@ impl From<BaseAddresses> for PointerChains {
             inf_consumables: bitflag!(0b1000; world_chr_man, 0x80, offs_no_goods_consume as _),
             deathcam: bitflag!(0b1; world_chr_man, offs_deathcam as usize),
             evt_draw: bitflag!(0b1; sprj_debug_event, 0xa8),
+            bloodstain_draw: bitflag!(0b1; world_chr_man, 0x40, 0x0, 0x58, 0x338, 0x0, 0x1dd),
```

Comment:
Works great in 1.08+, but not in 1.04. The pointer chain here is a bit long, I'd look for a shorter one but I'm not opposed to keeping this one (with the caveat) if we can't find it.

Considering that it also needs `debug_draw`, maybe we can find an offset from `base_hbd` of roughly the magnitude of `hurbox` that points to the same address?

Reply:
I tried a different base address, which appears to correspond to c0000 under WorldChrMan in the debug menu. Maybe that one works fine on 1.04? It also gets rid of the first offset in the pointer chain, which might help?